src/test/java/net/sf/jabref/logic/integrity/IntegrityCheckTest.java

```diff
@@ -72,6 +78,33 @@ public void testTitleChecks() {
     }
 
     @Test
+    public void testAbbreviationChecks() {
+        assertCorrect("Proceedings of the", IntegrityCheck.ABBREVIATION_CHECKER);
+        assertWrong("Proc. of the", IntegrityCheck.ABBREVIATION_CHECKER);
+    }
+
+    @Test
+    public void testFileChecks() {
+        MetaData metaData = Mockito.mock(MetaData.class);
+        Mockito.when(metaData.getFileDirectory("file")).thenReturn(Collections.singletonList("."));
+        BibDatabaseContext context = new BibDatabaseContext(new BibDatabase(), metaData, new Defaults());
+
+        assertCorrect(":build.gradle:gradle", IntegrityCheck.FILE_CHECKER, context);
+        assertWrong(":asflakjfwofja:PDF", IntegrityCheck.FILE_CHECKER, context);
+    }
+
+    @Test
+    public void testTypeChecks() {
+        LinkedList<IntegrityMessage> messages = new LinkedList<>();
+        IntegrityCheck.TYPE_CHECKER.check("11--15","pages",new BibEntry("asdfasdf", "inproceedings"), messages, new BibDatabaseContext());
+        assertEquals(Collections.emptyList(), messages);
+
```

Comment:
At the moment, we do have a nice parallel structure, as each integrity check has its own test method in a single test class. If I would split this one test, I would loose this structure. Hence, I think it would make matters worse.